components/StyledInputField.js

```diff
@@ -35,6 +44,9 @@ const StyledInputField = ({
   hideOptionalLabel,
   useRequiredLabel,
   isPrivate,
+  helpText,
+  helpIconSize,
+  helpIconColor,
```

Comment:
It makes sense to add this to `StyledInputField`. I would however remove these two props, the size and color of the icon should ideally be inferred from the parent text (you can use `size="1em"` of 0.9em for example)

Reply:
So you mean these two should be inferred the two props we already have, `labelFontSize` and `labelColor`? I've made that change. Let me know if that's in agreement. 🤔